packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,204 @@
+| description   | `string`, `JSX`                                             | `undefined` | Body content                                                                               | ✅       |
+| title         | `string`                                                    | `undefined` | A brief heading                                                                            |          |
+| isDismissable | `boolean`                                                   | `true`      | Shows a dismiss button                                                                     |          |
+| onDismiss     | `function`                                                  | `undefined` | Callback when the dismiss button is pressed                                                |          |
+| contrast      | `high`, `low`                                               | `low`       | Can be set to `high` for more prominent look _(not related to a11y)_                       |          |
+| isFullWidth   | `boolean`                                                   | `false`     | Spans the entire width of container, otherwise max width is restricted to 584px by default |          |
+| variant       | `info`, `positive`, `notice`, `negative`                    | `info`      | Sets the color tone of entire alert. Icon is set automatically based on variant            |          |
+| isBorderless  | `boolean`                                                   | `false`     | Removes borders and border radii, useful for creating a full bleed layout                  |          |
+| actions       | `{ primary: PrimaryAction, secondary: SecondaryAction }` \* | `{}`        | Renders a primary action button and a secondary action link button                         |          |
+
+`PrimaryAction` and `SecondaryAction` will accept objects with the following keys:
+
+### PrimaryAction
+
+Renders and accepts a subset of `Button` props as keys:
+
+| Key       | Type       | Default     | Description | Required |
+| --------- | ---------- | ----------- | ----------- | -------- |
+| `text`    | `string`   | `undefined` | Content     | ✅       |
+| `onClick` | `function` | `undefined` | Callback    |          |
+
+### SecondaryAction
+
+Renders and accepts a subset of `Link` (button) props as keys:
+
+| Key       | Type       | Default     | Description | Required |
+| --------- | ---------- | ----------- | ----------- | -------- |
+| `text`    | `string`   | `undefined` | Content     | ✅       |
+| `href`    | `string`   | `undefined` | Hyperlink   |          |
+| `onClick` | `function` | `undefined` | Callback    |          |
+
+There'll also be an internal `BaseAlert` component that handles rendering web or native version similar to other Blade components. API would be similar to `Alert`.
```

Comment:
Why do we need an internal `BaseAlert`? For Link & Button we needed `Base*` since there were certain props we didn't want to expose to the consumer. I can't see that use-case here 🤔